python/validator/schema.json

```diff
@@ -52,6 +52,14 @@
             "type": "string",
             "required": true
           },
+          "enforcement_org_unit_cd": {
+            "type": "string",
+            "required": true
+          },
+          "enforcement_org_unit_cd_txt": {
+            "type": "string",
+            "required": true
+          },
```

Comment:
Same for this. Not needed to change this file

Reply:
Same as above